Preview not working (npm run preview)

### Describe the bug

The following occurs after a completely fresh installation.

npm run preview

error when starting preview server:
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Volumes/.../zzz/.svelte-kit/output/server/internal.js' imported from /Volumes/.../zzz/node_modules/@sveltejs/kit/src/exports/vite/preview/index.js
    at new NodeError (node:internal/errors:399:5)
    at finalizeResolution (node:internal/modules/esm/resolve:232:11)
    at moduleResolve (node:internal/modules/esm/resolve:850:10)
    at defaultResolve (node:internal/modules/esm/resolve:1070:11)
    at DefaultModuleLoader.resolve (node:internal/modules/esm/loader:306:12)
    at DefaultModuleLoader.getModuleJob (node:internal/modules/esm/loader:155:32)
    at DefaultModuleLoader.import (node:internal/modules/esm/loader:265:12)
    at importModuleDynamically (node:internal/modules/esm/translators:107:35)
    at importModuleDynamicallyCallback (node:internal/modules/esm/utils:87:14)
    at preview (file:///Volumes/.../zzz/node_modules/@sveltejs/kit/src/exports/vite/preview/index.js:40:77)

### Reproduction

npm create svelte@latest zzz
cd zzz
npm install
npm run dev <-- this works
npm run preview <-- this fails with the error shared above.

### Logs

```Shell
npm run preview

> zzz@0.0.1 preview
> vite preview


error when starting preview server:
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Volumes/.../zzz/.svelte-kit/output/server/internal.js' imported from /Volumes/.../zzz/node_modules/@sveltejs/kit/src/exports/vite/preview/index.js
    at new NodeError (node:internal/errors:399:5)
    at finalizeResolution (node:internal/modules/esm/resolve:232:11)
    at moduleResolve (node:internal/modules/esm/resolve:850:10)
    at defaultResolve (node:internal/modules/esm/resolve:1070:11)
    at DefaultModuleLoader.resolve (node:internal/modules/esm/loader:306:12)
    at DefaultModuleLoader.getModuleJob (node:internal/modules/esm/loader:155:32)
    at DefaultModuleLoader.import (node:internal/modules/esm/loader:265:12)
    at importModuleDynamically (node:internal/modules/esm/translators:107:35)
    at importModuleDynamicallyCallback (node:internal/modules/esm/utils:87:14)
    at preview (file:///Volumes/.../zzz/node_modules/@sveltejs/kit/src/exports/vite/preview/index.js:40:77)
```


### System Info

```Shell
System:
    OS: macOS 13.4.1
    CPU: (8) x64 Intel(R) Core(TM) i7-8569U CPU @ 2.80GHz
    Memory: 22.07 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.0.0 - /usr/local/opt/nvm/versions/node/v20.0.0/bin/node
    npm: 9.6.7 - /usr/local/opt/nvm/versions/node/v20.0.0/bin/npm
    pnpm: 8.6.6 - ~/Library/pnpm/pnpm
  Browsers:
    Chrome: 114.0.5735.198
    Safari: 16.5.1
  npmPackages:
    @sveltejs/adapter-auto: ^2.0.0 => 2.1.0
    @sveltejs/kit: ^1.20.4 => 1.22.1
    svelte: ^4.0.0 => 4.0.5
    vite: ^4.3.6 => 4.4.1
```


### Severity

blocking all usage of SvelteKit

### Additional Information

_No response_

--- a/.changeset/shy-colts-sniff.md
+++ b/.changeset/shy-colts-sniff.md
@@ -1,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+feat: more helpful error for preview if build output doesn't exist

--- a/packages/kit/src/exports/vite/preview/index.js
+++ b/packages/kit/src/exports/vite/preview/index.js
@@ -37,7 +37,7 @@ export async function preview(vite, vite_config, svelte_config) {
 	const dir = join(svelte_config.kit.outDir, 'output/server');
 
 	if (!existsSync(dir)) {
-		throw new Error(`Server files not found at ${dir}, did you run \`build\` first?`)
+		throw new Error(`Server files not found at ${dir}, did you run \`build\` first?`);
 	}
 
 	/** @type {import('types').ServerInternalModule} */

--- a/packages/kit/src/exports/vite/preview/index.js
+++ b/packages/kit/src/exports/vite/preview/index.js
@@ -1,4 +1,4 @@
-import fs, { existsSync } from 'node:fs';
+import fs from 'node:fs';
 import { join } from 'node:path';
 import { pathToFileURL } from 'node:url';
 import sirv from 'sirv';
@@ -36,7 +36,7 @@ export async function preview(vite, vite_config, svelte_config) {
 
 	const dir = join(svelte_config.kit.outDir, 'output/server');
 
-	if (!existsSync(dir)) {
+	if (!fs.existsSync(dir)) {
 		throw new Error(`Server files not found at ${dir}, did you run \`build\` first?`);
 	}
 

--- a/packages/kit/src/exports/vite/preview/index.js
+++ b/packages/kit/src/exports/vite/preview/index.js
@@ -1,4 +1,4 @@
-import fs from 'node:fs';
+import fs, { existsSync } from 'node:fs';
 import { join } from 'node:path';
 import { pathToFileURL } from 'node:url';
 import sirv from 'sirv';
@@ -35,6 +35,10 @@ export async function preview(vite, vite_config, svelte_config) {
 	const etag = `"${Date.now()}"`;
 
 	const dir = join(svelte_config.kit.outDir, 'output/server');
+
+	if (!existsSync(dir)) {
+		throw new Error(`Server files not found at ${dir}, did you run \`build\` first?`)
+	}
 
 	/** @type {import('types').ServerInternalModule} */
 	const { set_assets } = await import(pathToFileURL(join(dir, 'internal.js')).href);

